src/format/opamFile.ml

```diff
@@ -823,10 +826,13 @@ module ConfigSyntax = struct
                (kind,criterion)::List.remove_assoc kind t.solver_criteria }
   let with_solver t solver = { t with solver = Some solver }
 
+  let backup_url_ocamlpro = "http://opam.ocamlpro.com/backup"
```

Comment:
I would rather use this value when initialising the original config file (at `opam init` time, in `OpamClient`) than set it in the "empty" data structure. This way it can be seen and removed by hand later on. An initial default rather than a permanent, hidden default.
Also, the printer for lists assumes that an empty field of type list is the empty list, so this may cause issues -- e.g. you can't actually unset the field, or it reappears if opam reprints the file.